package.json

```diff
@@ -79,13 +79,13 @@
     "solhint": "^1.1.8",
     "solidity-sha3": "^0.4.1",
     "tiny-promisify": "^1.0.0",
-    "truffle": "4.0.4",
-    "tslint-no-unused-expression-chai": "0.0.3",
+    "truffle": "4.0.3",
```

Comment:
interesting that truffle was downgraded

Reply:
This may have been accidental.  investigating.

Reply:
Bumped version back to 4.0.4